src/components/App/App.js

```diff
@@ -57,6 +57,17 @@ class App extends Component {
     this.mounted = false;
   }
 
+  signOut = () => {
```

Comment:
Method for signing a user out - updates state to remove previous user info and sets isLoggedIn back to false.